docs/en/stack/ml/anomaly-detection/anomaly-detection-scale.asciidoc

```diff
@@ -170,6 +170,11 @@ The statistics can be calculated in the job itself or via aggregations. It is
 more efficient to use an aggregation when it’s possible, as in this case, the 
 data node does the heavy-lifting instead of the {ml} node.
 
+You may want to use `chunking_config` to tune your search speed when your 
+{dfees} use aggregations. In these cases, set `chunking_config`.`mode` to 
```

Comment:
Should it be "`chunking_config`.`mode`" or "`chunking_config.mode`? I see we include the dot in the quotes below for the `search.max_buckets` limit.